### PR TITLE
Add actions repository variables auditor

### PR DESCRIPTION
**Pull request by tspascoal**

### CODE REVIEW

File: src/auditors/repository-actions-variables.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    repo,
+    per_page: 1
+  });
+
+  if (data.total_count > 0) {
+    return [
+      {
+        message: `This repository has ${pluralize(
+          data.total_count,
+          'action variable',
```
Comment (timrogers) [inline src/auditors/repository-actions-variables.ts:28]:
```suggestion
          'GitHub Actions variable',
```

File: src/auditors/repository-actions-variables.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+          'action variables',
```
Comment (timrogers) [inline src/auditors/repository-actions-variables.ts:29]:
```suggestion
          'GitHub Actions variables',
```

File: test/auditors/repository-actions-variables.test.ts [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        ]        
+      });
+
+    const auditorArguments = buildAuditorArguments({ fetchMock: fetch });
+    const warnings = await auditor(auditorArguments);
+
+    expect(warnings).toEqual([
+      {
+        message:
+          'This repository has 2 action variables, which will not be migrated',
```
Comment (timrogers) [inline test/auditors/repository-actions-variables.test.ts:28]:
```suggestion
          'This repository has 2 GitHub Actions variables, which will not be migrated',
```

File: test/auditors/repository-actions-variables.test.ts [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+      },
+    ]);
+  });
+
+  it("returns no warnings if there aren't any repository action variables", async () => {
+    const fetch = fetchMock
+      .sandbox()
+      .getOnce('https://api.github.com/repos/test/test/actions/variables?per_page=1', []);
```
Comment (timrogers) [inline test/auditors/repository-actions-variables.test.ts:36]: We're expecting this to always return a `total_count` in the code, right?

**Review by timrogers — CHANGES_REQUESTED** [review]
Thanks for the speedy contribution ❤️ I've got some copy tweaks to suggest, and I think this may not work.

File: test/auditors/repository-actions-variables.test.ts (hunk 4 above)
Comment (tspascoal) [reply test/auditors/repository-actions-variables.test.ts:36]: yes. Total count is independent of the number of results returned, so we don't have to paginate to get the number of variables.

**Pull request merged**